### DISCUSSION TITLE
Top Left / Top Right / Bottom Left / Bottom Right Thirds

### DISCUSSION THREAD

**Discussion by slifty** [rxhanson/Rectangle#724] [Ideas]
Rectangle currently supports sixths which is great -- it also supports vertically combining pairs of sixths (e.g. vertical thirds)

Could we allow HORIZONTAL combining of sixths as well?  This would result in a top left, top right, bottom left, bottom right thirds, allowing completely filled third / sixth layouts with only three windows

For instance this would allow layouts of three windows (A, B, C):

```
A B B
A C C
```

or

```
B B A
C C A
```

Would you consider a PR for this to add CLI support (akin to the ninths functionality)?

**Comment by rxhanson** [comment]
If you change the "repeated commands" setting to "cycle 1/2, 2/3, and 1/3 on half actions" or "move to adjacent on left/right, or cycle size on half", then you can achieve these layouts with the corner actions. 

But, I realize that this would require a lot more shortcut presses and mental juggling than if they were their own separate actions. Your ninths PR was good, so I would merge a PR for this from you if you wanted to add it in.

**Comment by slifty** [comment]
Opened a PR!  Thanks for considering the feature -- https://github.com/rxhanson/Rectangle/pull/740
